bin/logstash

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+set -x
```

Comment:
Is there a reason to not add `set -eu`?

Reply:
@afoerster I'll add that. Should that be added to the bin/log-collector script as well?